packages/main/src/themes/Dialog.css

```diff
@@ -46,19 +46,40 @@
 	position: relative;
 }
 
-:host([state="Error"]) .ui5-popup-header-root {
-	box-shadow: var(--_ui5_dialog_header_error_state_box_shadow);
+:host([state="Error"]) .ui5-popup-header-root::before,
+:host([state="Information"]) .ui5-popup-header-root::before,
+:host([state="Success"]) .ui5-popup-header-root::before,
+:host([state="Warning"]) .ui5-popup-header-root::before {
```

Comment:
Try `:host([state]:not([state="None"])) .ui5-popup-header-root::before {` instead. If you like it keep it.